packages/app/src/app/graphql/types.ts

```diff
@@ -1332,6 +1340,8 @@ export type CurrentTeamInfoFragmentFragment = { __typename?: 'Team' } & Pick<
     invitees: Array<
       { __typename?: 'User' } & Pick<User, 'id' | 'avatarUrl' | 'username'>
     >;
+    userAuthorizations: Array<MemberAuthorization>;
```

Comment:
```suggestion
    userAuthorizations: MemberAuthorization[];
```

Reply:
New to TS, what's the difference between the two?

Reply:
There’s no real difference but in general `Type[]` is used always when you have so called “simple types” (ie a named type), `Array<{}>` is used whenever you haven’t defined a type separately as in `Array<{a: string; b: number}>`